### PR TITLE
WIP: Feat: upgrade to webnative 0.35.0

### PR DESCRIPTION
**Pull request by avivash**
# Description

- Updating this template to use `webnative` 0.35.0
- Adding UI refresh
- Adding recovery kit flow
- Adding Theme preference updates

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [x] Refactor (non-breaking change that updates existing functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by icidasset** [comment]
That's... interesting 🤔 It's as if the whole FileSystem object became read-only/frozen somehow.

It might have to do something with that `recoil` library you're using?
It looks like it freezes objects in dev?
https://github.com/facebookexperimental/Recoil/blob/dfb05b9c7c77c8adcea2a6d71504b4063ef1a784/packages/recoil/recoil_values/Recoil_atom.js#L216-L231

**Comment by avivash** [comment]
> That's... interesting 🤔 It's as if the whole FileSystem object became read-only/frozen somehow.
> 
> It might have to do something with that `recoil` library you're using? It looks like it freezes objects in dev? https://github.com/facebookexperimental/Recoil/blob/dfb05b9c7c77c8adcea2a6d71504b4063ef1a784/packages/recoil/recoil_values/Recoil_atom.js#L216-L231

ahh good find! interesting... i'll take a look, thanks!

File: src/stores.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { loadTheme, type THEME } from "./lib/theme";
 export const filesystemStore = atom({
   key: "filesystem",
   default: null as FileSystem | null,
+  dangerouslyAllowMutability: true,
```
Comment (avivash) [inline src/stores.ts:12]: @icidasset i was able to find this option to disable `object.freeze` in development mode. Thanks for finding that function! 😄

File: src/App.tsx [hunk 2]
```diff
@@ -19,6 +20,42 @@ import Register from './routes/RegisterRoute';
 import Settings from './routes/SettingsRoute';
 import NotFound from './routes/NotFoundRoute';
 
+const AppRenderer = () => {
+  const location = useLocation();
+  const session = useRecoilValue(sessionStore);
+
+  if (session.loading && !location.pathname.match(/register|backup|delegate/)) {
```
Comment (avivash) [inline src/App.tsx]: I'll look into removing the requirement for checking the path here 👍🏼

File: src/lib/init.ts [hunk 3]
```diff
@@ -1,61 +1,58 @@
...
-setup.debug({ enabled: false });
 
 const initialize = async (): Promise<void> => {
   try {
     let backupStatus: BackupStatus = null;
 
-    const state: webnative.AppState = await webnative.app({ useWnfs: true });
+    const program: webnative.Program = await webnative.program({
+      tag: { creator: "Fission", name: "WAT" },
+      debug: process.env.NODE_ENV === 'development',
```
Comment (avivash) [inline src/lib/init.ts]: now tying `debug` directly to development mode. CRA ensures this is always exposed

**Pull request merged**